### DISCUSSION TITLE
Enemies for normal mode

### DISCUSSION THREAD

**Discussion by deltasolidsnake** [WizardMaster38/Infernum-Ad-Infinitum-To-Hell-and-Beyond#13] [Ideas]
The following comments will be the suggestions.

Don't worry about in-text representation, pics are just for reference.

**Comment by deltasolidsnake** [comment]
_**Slime**_
Slimes are low level monsters that are used through the whole game as they are adaptable.
Slimes can attack by throwing objects, throwing themselves, or trying to eat you.
Later in the game, slimes can mimic players, morphing into them and mimicking abilities. This can only be done if the slime has successfully damaged the player once.
![502a858e852dde0d7602e07fa9ee23ed](https://user-images.githubusercontent.com/88522032/128608130-50d27f89-e005-4850-81d1-c88cd059de70.jpg)

**Reply by WizardMaster38** [reply]
cute! different variations sound like a good idea, will be added

**Comment by deltasolidsnake** [comment]
_**Possessed armor**_
Possessed armor is a mid-level monster it is found in darker areas and at night.
weak to blunt and magic damage.
![636306156895834255](https://user-images.githubusercontent.com/88522032/128608393-1860d826-2bcd-4b21-bcef-da44e231c551.jpeg)
![1453955016 croxot_armor_monster_girl](https://user-images.githubusercontent.com/88522032/128608394-e0d082a1-959f-4d3e-9792-1097f9decfa2.png)

**Reply by WizardMaster38** [reply]
if it's weak to both blunt and magic damage, how do you damage it? 🤔

**Reply by deltasolidsnake** [reply]
Breaking the armor itself.

**Reply by deltasolidsnake** [reply]
Magic kills the ghost, blunt breaks the armor.

**Reply by WizardMaster38** [reply]
wait i'm stupid
what is it strong against

**Reply by deltasolidsnake** [reply]
Sharp(?) damage, fire damage, and poison damage.

**Reply by deltasolidsnake** [reply]
It also deals the damage with whatever weapon it's holding.

**Reply by WizardMaster38** [reply]
Ok sounds good
